gqcp/tests/Operator/SecondQuantized/USQOneElectronOperator_test.cpp

```diff
@@ -188,4 +190,119 @@ BOOST_AUTO_TEST_CASE ( calculateExpectationValue_throw ) {
     BOOST_CHECK_THROW(h.calculateExpectationValue(D_valid, D_invalid), std::invalid_argument);
 
     BOOST_CHECK_NO_THROW(h.calculateExpectationValue(D_valid, D_valid));
-}
\ No newline at end of file
+}
+
+
+/**
+ * Check whether or not calculateExpectationValue shows the correct behaviour
+ */
+BOOST_AUTO_TEST_CASE ( calculateExpectationValue_behaviour ) {
+    
+    const size_t dim = 2;
+
+    // Initialize a test matrix and convert it into an operator
+    GQCP::QCMatrix<double> M1 (dim);
+    M1 << 1.0, 2.0,
+          3.0, 4.0;
+    const GQCP::ScalarUSQOneElectronOperator<double> op (M1, M1);
+
+    // initialize an alpha and beta density matrix, each one is chosen as a hermitian matrix.
+    GQCP::QCMatrix<double> d_alpha (dim);
+    d_alpha << 0.0, 1.0,
+               1.0, 0.0;
```

Comment:
```suggestion
    GQCP::QCMatrix<double> D_alpha (dim);
    D_alpha << 0.0, 1.0,
               1.0, 0.0;
```

The 1-DM is usually given the symbol `D` instead of `d`.